WORKING_GROUPS.md

```diff
@@ -434,6 +435,37 @@ Responsibilities include:
   backporting changes to these branches.
 * Define the policy for what gets backported to release streams.
 
+### [Security](https://github.com/nodejs/security-wg)
+
+The Security Working Group manages all aspects and process linked to security for Node.js.
```

Comment:
process -> processes

... and maybe...

security for Node.js -> Node.js security